pkgs/development/python-modules/swagger-ui-bundle/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildPythonPackage, fetchPypi, jinja2, flake8 }:
+
+buildPythonPackage rec {
+  pname = "swagger-ui-bundle";
+  version = "0.0.2";
+
+  src = fetchPypi {
+    pname = "swagger_ui_bundle";
+    inherit version;
+    sha256 = "0kk6y8p1fhmnjnxfafdw6dm1xc5wj8101y5h7hcy3rb2bxc1i94q";
+  };
+
+  nativeBuildInputs = [ flake8 ];
```

Comment:
I'm seeing

```
setup_requires=['pytest-runner', 'flake8'],
```

Which is only useful for running tests, and we don't have any so it's just bloating the build closure.

I usually propose to patch this away.